src/ui/server/doc/openapi_v1.json

```diff
@@ -77,6 +77,56 @@
           }
         }
       }
+    },
+    "/Statistic": {
+      "get": {
+        "operationId": "getNumberOfIsolates",
+        "tags": [
+          "Util"
+        ],
+        "summary": "Retrieve the staistic about the dataset",
+        "description": "Returns object with the total number of isolaets and statistic table",
+        "responses": {
+          "200": {
+            "description": "successful operation",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/Statistic"
+                }
+              }
+            }
+          },
+          "500": {
+            "$ref": "#/components/responses/Standard500ErrorResponse"
+          }
+        }
+      }
+    },
+    "/selectedSettings": {
+      "put": {
```

Comment:
I'm having second thoughts about PUT vs POST...I can't remember why one was chosen over the other in the MiBi portal.  Can you ask Isaak if he remembers the reasoning behind our decision.

Reply:
POST is used to create new record and PUT to update the complete existing record. But there is also PATCH. It is used for partial updates. So only a part of the resource is changed and the rest is kept. So I thought about using PATCH because only one filter is changed with one click and the other filters stay the same. But I will ask Isaak today in the stand up what he thinks about it.